### ISSUE TITLE
Tag all pre-releases as release candidates

### ISSUE THREAD

**Issue by eproxus** [erlang/docker-erlang-otp#470]
Can pre-release canditates always be tagged with `rc`? I'm thinking of the major version tags like `26` or `27`. Currently, if one wants to stay always on the latest stable major version, it is not possible to tell if `27` is stable or not.

In this case I think `27` should be tagged `27-rcX` instead until a stable version is released.

**Comment by getong** [comment]
As in https://hub.docker.com/_/erlang , 

[27.0-rc3.0.0, 27.0-rc3.0, 27.0-rc3, 27](https://github.com/erlang/docker-erlang-otp/blob/4aa0cf2537dc2371bbd51a378695eba30637e345/27/Dockerfile)
[27.0-rc3.0.0-slim, 27.0-rc3.0-slim, 27.0-rc3-slim, 27-slim](https://github.com/erlang/docker-erlang-otp/blob/4aa0cf2537dc2371bbd51a378695eba30637e345/27/slim/Dockerfile)
[27.0-rc3.0.0-alpine, 27.0-rc3.0-alpine, 27.0-rc3-alpine, 27-alpine](https://github.com/erlang/docker-erlang-otp/blob/4aa0cf2537dc2371bbd51a378695eba30637e345/27/alpine/Dockerfile)

Anything confused?

**Comment by eproxus** [comment]
Yes, I'd like the tag `27` to be `27-rc3` so that it is clear it is not a final release yet.

**Comment by getong** [comment]
There are four tags, you can use 27.0-rc3 now.

**Comment by eproxus** [comment]
My issue is not with `27.0-rc3`, the problem is (in my opinion) that `27` exists at all. Since it gives the impression that it is stable.
